### PR TITLE
chore: Update to Astro 2.0

### PR DESCRIPTION
**Pull request by bai**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Happy Monday! Update the website to Astro 2.0 🙏🏼

### CODE REVIEW

File: www/astro.config.mjs [hunk 1]
```diff
@@ -43,7 +42,6 @@ export default defineConfig({
       theme: "rose-pine",
       wrap: true,
     },
-    extendDefaultPlugins: true,
```
Comment (bai) [inline www/astro.config.mjs:46]: This is now the default.

File: www/astro.config.mjs [hunk 2]
```diff
@@ -8,11 +8,10 @@ import remarkCodeTitles from "remark-code-titles";
...
 import rehypeAutolinkHeadings from "rehype-autolink-headings";
 import sitemap from "@astrojs/sitemap";
-/**
- * @link https://astro.build/config
- */
+
+/** @link https://astro.build/config */
 export default defineConfig({
-  site: `https://create.t3.gg`,
+  site: `https://create.t3.gg/`,
```
Comment (bai) [inline www/astro.config.mjs:14]: Per Astro upgrade guide: https://docs.astro.build/en/guides/upgrade-to/v2/#changed-astrosite-trailing-slash

**Review by c-ehrlich — CHANGES_REQUESTED** [review]
Upgrade to Astro 2.0 looks good. Can add content collections in the future, but even without them it's a good upgrade.

Could you move the changes to the CLI to a separate PR, and delete the changeset?

**Comment by bai** [comment]
@c-ehrlich Sure thing, done 🙏🏼

**Comment by bai** [comment]
Looks like one tiny change to the CLI has to stay to make eslint happy.

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**